### PR TITLE
fix(DB/creature): correct spawn position of npc 1568

### PR DESCRIPTION
**Pull request by schell244**
## Changes Proposed:
Correct spawn position of npc Undertaker Mordo (id: 1568) 
https://classic.wowhead.com/npc=1568/undertaker-mordo

Unbenannt

## Issues Addressed:
- Closes 

## SOURCE:
- VMangos DB

## Tests Performed:
- tested in game

## How to Test the Changes:
1. create new undead character
2. check spawn position of Undertaker Mordo

## Known Issues and TODO List:
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
Can confirm from the TBCC PTR.

```
[6] MoverGUID: Full: 0x204CB0000001880000003300004E452A Creature/0 R4908/S51 Map: 0 Entry: 1568 Low: 5129514
[6] Movement Flags: 0 (None)
[6] Movement Flags 2: 0 (None)
[6] Movement Flags 3: 0 (None)
[6] MoveTime: 822055483
[6] Position: X: 1678.9888 Y: 1667.8617 Z: 135.8549
[6] Orientation: 3.769911050796508789
```

**Comment by acidmanifesto** [comment]
old retail sniff:
correct sniff position from wotlk retail:
```
[6] MoverGUID: Full: Ë0äR Creature/0 R4908/S51 Map: 0 Entry: 1568 Low: 6584259
[6] Movement Flags: 0 (None)
[6] Movement Flags 2: 0 (None)
[6] Movement Flags 3: 0 (None)
[6] MoveTime: 822055483
[6] Position: X: 1671.89 Y: 1678.29 Z: 120.802
[6] Orientation: 0.0349066
```

**Comment by Nyeriah** [comment]
@schell244 thanks for the PR, but it seems indeed this is incorrect, as many NPCs in starting areas were moved to be closer to the player's spawn position, which seems to be the case here

File: data/sql/updates/pending_db_world/rev_1658058502913230900.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- Undertaker Mordo https://classic.wowhead.com/npc=1568/undertaker-mordo
+UPDATE `creature` SET `position_x` = 1678.99,`position_y` = 1667.86,`position_z` = 135.855, `orientation` = 3.76991 WHERE `id1` = 1568;
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658058502913230900.sql:2]: I still think this review is still valid, just readjust it to the sniff parse i posted in the commits and resubmit the pr
~Sniff Team

UPDATE `creature` SET `position_x` = 1671.89,`position_y` = 1678.29,`position_z` = 120.802, `orientation` = 0.0349066 WHERE `id1` = 1568;

Comment (schell244) [reply data/sql/updates/pending_db_world/rev_1658058502913230900.sql:2]: Thanks, I've adapted and reopened the PR.

**Pull request closed without merging**